bootloader/hos/hos.c

```diff
@@ -623,13 +626,16 @@ int hos_launch(ini_sec_t *cfg)
 		if (!exoFwNumber)
 			exoFwNumber = 4;
 	case KB_FIRMWARE_VERSION_500:
+		if (!exoFwNumber)
+			exoFwNumber = 4;
```

Comment:
exoFwNumber = 5 for KB_FIRMWARE_VERSION_500

Reply:
fixed